inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -248,28 +252,26 @@ def _send_robotupload(obj, eng):
         marc_json = marcxml_processor.do(data)
         marcxml = legacy_export_as_marc(marc_json)
 
-        if not url and current_app.debug:
-            # Log what we would send
-            current_app.logger.debug(
-                "Going to robotupload {mode} to {url}:\n{marcxml}\n".format(
-                    url=url,
-                    marcxml=marcxml,
-                    callback_url=combined_callback_url,
-                    mode=mode,
-                    nonce=obj.id,
-                    priority=5,
-                )
-            )
-            return
-        else:
-            result = make_robotupload_marcxml(
+        if current_app.debug:
```

Comment:
But in principle we could connect them, by simply tuning the logger depending on the current_app.debug flag. In this way your code doesn't have to check all the times if you are in debug mode or not. It's the logger that decides how to filter.

I think we should leverage on `invenio-logging`:
https://github.com/inveniosoftware/invenio-logging/blob/master/invenio_logging/fs.py#L46